### PR TITLE
[omplapp] vendors own ompl

### PR DESCRIPTION
**Pull request by autoantwort**
This does not change the status quo. I don't want to change the status quo and iiuc devendoring ompl out of omplapp is a larger effort.

### CODE REVIEW

File: ports/omplapp/portfile.cmake [hunk 1]
```diff
@@ -77,7 +77,7 @@ if(VCPKG_LIBRARY_LINKAGE STREQUAL "static")
     file(REMOVE_RECURSE "${CURRENT_PACKAGES_DIR}/bin" "${CURRENT_PACKAGES_DIR}/debug/bin")
 endif()
 
-if(NOT VCPKG_TARGET_IS_WINDOWS)
+if(EXISTS "${CURRENT_PACKAGES_DIR}/lib/pkgconfig/ompl.pc")
```
Comment (Neumann-A) [inline ports/omplapp/portfile.cmake:80]:
```suggestion
```

Why do the check ?

Comment (autoantwort) [reply ports/omplapp/portfile.cmake:80]: Because there is no .pc file on some systems

Comment (Neumann-A) [reply ports/omplapp/portfile.cmake:80]: pc files should be installed on all systems.

Comment (autoantwort) [reply ports/omplapp/portfile.cmake:80]: You can create a bug report upstream

**Pull request closed without merging**